Dockerfile

```diff
@@ -39,6 +39,5 @@ COPY --from=hermetic /workspace/go.sum go.sum
 COPY --from=hermetic /workspace/cmd cmd
 COPY --from=hermetic /workspace/doc doc
 COPY --from=hermetic /workspace/examples examples
-COPY --from=hermetic /workspace/hack hack
 COPY --from=hermetic /workspace/pkg pkg
-COPY --from=hermetic /workspace/vendor vendor
\ No newline at end of file
+COPY --from=hermetic /workspace/vendor vendor2
```

Comment:
Intentional?

Reply:
Yes - I noticed that the `COPY` command wouldn't completely overwrite the local directory (e.g. if some files were deleted in the docker build stage, those deletions wouldn't be mirrored in the destination context). Copying to a different directory and subsequently replacing it works, albeit a bit hacky. I changed the name to `vendor.tmp` to make it more obvious.